tests/unit_tests/expected_md/summary_regressions_baselineisnotparent.md

```diff
@@ -0,0 +1,13 @@
+Contender commit `abc` had 2 regressions compared to its baseline commit.
+
+### Benchmarks with regressions:
+
+- Run ID [some_contender1](https://conbench/some_baseline1...some_contender1)
```

Comment:
Is this the real URL form that is produced? or is this mocked? Mostly wondering if the endpoint should be `conbench/compare/runs/...`

Reply:
This was mocked for line length. The actual link does look like [this](https://github.com/conbench/benchalerts/pull/17/files#diff-6d0b260115c992883e4117c4e3d948b3fbe400a9605c380a50a00ce01bba2f3dR5)